--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -148,6 +148,7 @@
 install(FILES   media/texture_box.png       DESTINATION share/nodeboxeditor/media)
 install(FILES   media/texture_terrain.png   DESTINATION share/nodeboxeditor/media)
 install(FILES   media/icon.png              DESTINATION share/nodeboxeditor/media)
+install(FILES   media/icon.png              DESTINATION share/icons/hicolor/128x128/apps RENAME nodeboxeditor.png)
 install(FILES   media/rotate_x.png          DESTINATION share/nodeboxeditor/media)
 install(FILES   media/rotate_y.png          DESTINATION share/nodeboxeditor/media)
 install(FILES   media/rotate_z.png          DESTINATION share/nodeboxeditor/media)
--- a/nodeboxeditor.desktop
+++ b/nodeboxeditor.desktop
@@ -5,7 +5,7 @@
 GenericName[fr]=Éditeur de node boxes pour Minetest
 Comment[fr]=Créer et éditer des node boxes pour Minetest
 Exec=nodeboxeditor
-Icon=/usr/local/share/nodeboxeditor/media/icon.png
+Icon=nodeboxeditor
 Terminal=false
 Type=Application
 Categories=Development;
