--- nodeboxeditor-0.9.0.orig/CMakeLists.txt
+++ nodeboxeditor-0.9.0/CMakeLists.txt
@@ -156,7 +156,7 @@ install(FILES   media/flip_y.png
 install(FILES   media/flip_z.png          DESTINATION share/nodeboxeditor/media)
 install(FILES   editor.conf.example         DESTINATION share/nodeboxeditor)
 if(UNIX)
-	install (FILES   nodeboxeditor.desktop       DESTINATION ~/.local/share/applications)
+	install (FILES   nodeboxeditor.desktop       DESTINATION share/applications)
 endif()
 install(FILES   README.md                   DESTINATION share/nodeboxeditor)
 install(TARGETS ${PROJECT_NAME}             DESTINATION bin)
