--- nodeboxeditor-0.9.0.orig/src/main.cpp
+++ nodeboxeditor-0.9.0/src/main.cpp
@@ -40,6 +40,13 @@ void findWorkingDirectory(bool &editor_i
 		editor_is_installed = true;
 		return;
 	}
+
+	chdir("/usr/share/nodeboxeditor");
+	if (FileExists("media/sky.jpg")) {
+		std::cerr << "Is installed!";
+		editor_is_installed = true;
+		return;
+	}
 
 	char buff[PATH_MAX];
 	ssize_t len = ::readlink("/proc/self/exe", buff, sizeof(buff) - 1);
